libraries/rush-sdk/src/index.ts

```diff
@@ -126,51 +131,78 @@ if (sdkContext.rushLibModule === undefined) {
     const rushJson: JsonObject = JsonFile.load(rushJsonPath);
     const { rushVersion } = rushJson;
 
-    const installRunNodeModuleFolder: string = path.join(
-      monorepoRoot,
-      `common/temp/install-run/@microsoft+rush@${rushVersion}`
-    );
-
     try {
-      // First, try to load the version of "rush-lib" that was installed by install-run-rush.js
-      terminal.writeVerboseLine(`Trying to load  ${RUSH_LIB_NAME} installed by install-run-rush`);
-      sdkContext.rushLibModule = requireRushLibUnderFolderPath(installRunNodeModuleFolder);
-    } catch (e1) {
-      let installAndRunRushStderrContent: string = '';
-      try {
-        const installAndRunRushJSPath: string = path.join(monorepoRoot, 'common/scripts/install-run-rush.js');
+      const { RushGlobalFolder } = require('@microsoft/rush-lib/lib-esnext/api/RushGlobalFolder');
+      terminal.writeLine(RushGlobalFolder);
+      terminal.writeVerboseLine(`Try to load ${RUSH_LIB_NAME} from rush global folder`);
+      const rushGlobalFolder: typeof RushGlobalFolder = new RushGlobalFolder();
+      // The path needs to keep align with the logic inside RushVersionSelector
```

Comment:
Can that logic just be referenced here?

Reply:
Possible.

But the logic is located in `rush` instead of `rush-lib`.  https://github.com/microsoft/rushstack/blob/8ed5cea52f35a93da97b54a8b7e0465be4b2f34c/apps/rush/src/RushVersionSelector.ts#L31

By directly referencing, `rush-sdk` needs to depends on `rush` as well.